src/app/Interface/UserInterface.ts

```diff
@@ -123,6 +123,7 @@ export interface MemberData{
     TeamManager: boolean;
     Teams: [];
     Email: string;
+    OrgMember: string[];
 }
```

Comment:
what is the use of orgMember array in member data interface?

Reply:
no there is no use as such now bhaiya I removed this property.